### PR TITLE
Iterate on aspects

### PR DESCRIPTION
**Pull request by charles-uno**
No rules changes. Just tightening up what we have, trying to get back into it

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -83,38 +80,23 @@ We would also like to thank [Evil Hat][evil_hat] for inspiring our rule system.
...
-Before you can tell a story, you need a setting and some characters.
-This is a collaborative process that involves everyone at the table.
-Creating Aspects is more of an art than a science;
-there is no right or wrong way to do it as long as everyone is happy in the end.
-
-It can take some time to get the setting and characters figured out, so it's best to dedicate your first session to it.
-This also gives you some time after your first session to let the ideas you came up with develop.
+Before jumping into your adventure, you need a setting and some characters.
+In {GameName}, that means writing down a handful of aspects.
+This chapter explains what aspects are and how to use them to lay the foundation for your story.
```
Comment (agude) [inline _chapters/00-outline.md:85]: Like the intro to aspects

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -83,38 +80,23 @@ We would also like to thank [Evil Hat][evil_hat] for inspiring our rule system.
...
-They are always true; when you create an Aspect you establish a fact about the world.
-If an Aspect stops being true, remove it or rewrite it.
-
-Aspects keep everyone on the same page about what is happening in the story and what is important.
-When you write down an Aspect you don't have to have every implication of it figured out.
-Exploring and expanding the world together as you play is a key part of {AspectRPG}.
-By letting other players fill in details with their own Aspects, the players can collaboratively build the game world and lore.
+Aspect are short, improvised phrases written down for the whole table to see. 
+They are central to {GameName} in two ways:
+1. They keep everyone on the same page. If it's written down as an aspect, it's true and important. If an aspect is no longer true and important, update it or erase it.
```
Comment (agude) [inline _chapters/00-outline.md:91]: Do we need... To talk about "changing them"? This is sort of the only place we talk about "if it's not true just rewrite it!".

Maybe an example?

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -158,14 +140,13 @@ That means:
 - The PC's player starts each session with 2 fate points. 
 
 As PCs level up, these numbers all scale.
-Character advancement is discussed in the next chapter.
-
+Character advancement is discussed in the next chapter (as are rolls, fate points, and stress boxes).
 
 ```
-TODO: inline character sheet or link to one
+TODO: link to character sheet
```
Comment (agude) [inline _chapters/00-outline.md:146]:
```
{% link character_sheet.md %}
```

Will just work now.

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -186,31 +167,27 @@ I think we should probably err on the side of brevity. The examples below are pr
...
 > | Trouble: | `Lost My Boat in a Game of Cards`    |
 >
-> The second PC is Naka:
->
-> Captain Naka is a half-orc pyromancer (and pyromaniac) exiled by her magic-fearing clan.
-> She grew up on the streets of Three Bridges, where her quick tongue and speedy reflexes kept her safe from the worst consequences of her own actions.
-> A loner with trust issues, she now finds herself conflicted by her responsibility for {BoatName} and its crew.
+> Barry's PC is Naka of Three Bridges: a streetwise misanthrope with the power to breathe fire.
+> A series of suspicious "retirements" have recently left her at the helm of the {BoatName}. 
+> She now finds herself responsible for (and surrounded by) a crew of bloodthirsty pirates.
```
Comment (agude) [inline _chapters/00-outline.md:181]: Are the pirates really `bloodthirsty`? Seems to conflict with TBY's `Speak softly`.

**Review by agude — APPROVED** [review]
Some comments, but looks good!

File: _chapters/00-outline.md (hunk 2 above)
Comment (charles-uno) [reply _chapters/00-outline.md:91]: I'm inclined to leave this terse and call back to it later with an example. Examples feel clunky prior to setting and character creation

**Pull request merged**